Tarea 5: Rodolfo

rodoart opened the pull request.

rodoart committed:
Merge remote-tracking branch 'refs/remotes/lbenet/master'

rodoart committed:
tarea0

Aquí está mi pull request

rodoart committed:
Ya hice git add, git commit.

Espero tus modificaciones Uriel.

Gracias

UriAceves committed:
Agrego linea

rodoart committed:
Merge branch 'master' of https://github.com/rodoart/2017-1_TSFisComp-1

Mezclando archivos.

rodoart committed:
Merge pull request #1 from lbenet/master

Esto es una prueba

rodoart committed:
Merge pull request #2 from lbenet/master

Actualización online (Sin consola 2)

rodoart committed:
Merge pull request #3 from lbenet/master

Actualizacion 2

rodoart committed:
Merge pull request #4 from lbenet/master

Actualización

rodoart committed:
Merge pull request #5 from lbenet/master

Actualizacion

rodoart committed:
Merge pull request #6 from lbenet/master

Notebook 08, Tarea3, Tarea4 y errores en el notebook 07 (#60)

rodoart committed:
Merge pull request #7 from lbenet/master

Actualización

rodoart committed:
Merge pull request #8 from lbenet/master

Tarea2 (#38)

rodoart committed:
Merge pull request #9 from lbenet/master

Actualizacion

rodoart committed:
Archivo para hacer carpeta vacía.

rodoart committed:
Archivo de la tarea.

rodoart committed:
Lamento el retraso, pero tuve unos problemas.

Comento los problemas con el profe.

UriAceves commented:
En el ejercicio uno creo que te estás concentrando en cosas un poco diferentes de las que queremos, podrías ver por ejemplo cuánto vale el exponente de Lyapunov en donde hay bifurcaciones.

En el 2 y el 3 haces cosas pero no comentas nada.

Espero el 4 y el 5. Cuando tengas una nueva versión pon comentarios nuevos aquí.

rodoart committed:
Tarea 5 Rodolfo

Completé todo lo que me faltaba e incorporé la diferenciación automática.

rodoart committed:
Nueva versión, errores en fórmulas corregidos.

La otra versión no podía ejecutarse completamente, debido a un función que borre accidentalmente, esta versión ya corre.

UriAceves commented:
Listo, me parece bien así como está.

Hago el merge.
